### PR TITLE
Added support for spanish

### PR DESCRIPTION
**Pull request by ra1nbowpill**

### CODE REVIEW

**Comment by sindresorhus** [comment]
This looks identical to the English list.

File: data/es.json [hunk 1]
```diff
@@ -0,0 +1,816 @@
...
+
+	pokemon_names = [];
+	trs = document.getElementsByTagName("table")[0].children[1].children;
+	for (var i in trs) {
+		if (trs[i].children) {
+			pokemon_names.push(trs[i].children[1].children[0].text);
+		}
+	}
+	console.log(JSON.stringify(pokemon_names)) 
+*/
```
Comment (sindresorhus) [inline data/es.json]: You can't have comments in JSON.

**Comment by ra1nbowpill** [comment]
Here is the diff between en and es [diff_en_fr.txt](https://github.com/sindresorhus/pokemon/files/999677/diff_en_fr.txt) (I agree there's not much differences)

**Comment by sindresorhus** [comment]
Some of the items in the diff just looks like typos. I'm not really confident this is correct.


Seems like there are no official Spanish translations:
- http://www.serebiiforums.com/showthread.php?97444-Spanish-Pokemon-Names&s=d3e6c9c1fe965e774cfb2633cb93e5e2
- http://bulbapedia.bulbagarden.net/wiki/User:Raltseye/List_of_Spanish_Pok%C3%A9mon_names

**Pull request closed without merging**